.github/workflows/dev_on_pull_request_run_e2e.yml

```
@@ -47,29 +56,49 @@ jobs:
   e2e-tests-ios:
     if: |
       github.event.label.name == 'e2e iOS' ||
-      github.event.label.name == 'e2e'
+      github.event.label.name == 'e2e' ||
+      github.event.label.name == 'e2e:activation' ||
+      github.event.label.name == 'e2e:decouverte' ||
+      github.event.label.name == 'e2e:conversion'
     needs: notify_test_launch
     concurrency:
       group: e2e-ios-${{ github.head_ref }}
       cancel-in-progress: false
     uses: ./.github/workflows/dev_on_workflow_call_e2e_ios.yml
     with:
       upload_name_prefix: '[PR]'
+      test_tags: >-
+        ${{
+          github.event.label.name == 'e2e:activation' && 'activation' ||
+          github.event.label.name == 'e2e:decouverte' && 'decouverte' ||
+          github.event.label.name == 'e2e:conversion' && 'conversion' ||
+          'nightlyIOS'
+        }}
     secrets:
       GCP_EHP_SERVICE_ACCOUNT: ${{ secrets.GCP_EHP_SERVICE_ACCOUNT }}
       GCP_EHP_WORKLOAD_IDENTITY_PROVIDER: ${{ secrets.GCP_EHP_WORKLOAD_IDENTITY_PROVIDER }}
 
   e2e-tests-android:
     if: |
       github.event.label.name == 'e2e Android' ||
-      github.event.label.name == 'e2e'
+      github.event.label.name == 'e2e' ||
+      github.event.label.name == 'e2e:activation' ||
+      github.event.label.name == 'e2e:decouverte' ||
+      github.event.label.name == 'e2e:conversion'
     needs: notify_test_launch
     concurrency:
       group: e2e-android-${{ github.head_ref }}
       cancel-in-progress: false
     uses: ./.github/workflows/dev_on_workflow_call_e2e_android.yml
     with:
       upload_name_prefix: '[PR]'
+      test_tags: >-
+        ${{
+          github.event.label.name == 'e2e:activation' && 'activation' ||
+          github.event.label.name == 'e2e:decouverte' && 'decouverte' ||
+          github.event.label.name == 'e2e:conversion' && 'conversion' ||
+          'nightlyAndroid'
```

que penserais-tu de couper le début plutot que de faire du cas par cas ?

```suggestion
          github.event.label.name.startsWith('e2e:') ?
            github.event.label.name.replace('e2e:', '') :
            'nightlyAndroid'
```

---

J'ai fait un truc inspiré de ça dans mon prochain commit, assez générique pour que peu importe le label si on a les mots clés `activation`, `decouverte` ou `conversion` on puisse exécuter le scope voulu